app/assets/javascripts/spree/backend/solidus_multi_domain.js

```diff
@@ -0,0 +1,3 @@
+// Placeholder manifest file.
```

Comment:
We can remove empty files

Reply:
It has a require_tree at the bottom that will add `store_picker.js` as a dependency

Reply:
Can't we remove this `require_tree` statement?